source/fundamentals/indexes.txt

```diff
@@ -144,13 +150,47 @@ created above.
    :language: js
    :start-after: begin-query
    :end-before: end-query
+   :dedent:
 
 Multikey indexes behave differently from non-multikey indexes in terms of
 query coverage, index bound computation, and sort behavior. For a full
 explanation of multikey indexes, including a discussion of their behavior
 and limitations, refer to the :manual:`Multikey Indexes page
 </core/index-multikey>` in the MongoDB manual.
 
+Clustered Indexes
+~~~~~~~~~~~~~~~~~
+
+**Clustered indexes** are indexes that improve the performance of
+insert, update, and delete operations. When you create a clustered
+index, you also create a :v5.3:`clustered collection
+</core/clustered-collections/#std-label-clustered-collections>`. 
+
+To create a clustered index, specify the ``clusteredIndex`` option in
+the ``CollectionOption`` when you create a collection. The
```

Comment:
Issue: Sentence structure implies that the index creates the collection
Suggestion: "When you create a collection, specify the clusteredIndex option in the CollectionOption option"

Reply:
With the updates I made to the previous paragraph, I don't think the exact suggestion is needed. But, I'll remove "... when you create a collection" to avoid confusion.